pkgs/development/python-modules/pytest-ruff/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  attrs,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  poetry-core,
+  poetry-dynamic-versioning,
+
+  # dependencies
+  pytest,
+  ruff,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-ruff";
+  version = "0.4.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "pytest_ruff";
+    hash = "sha256-LJow8V84TCKciBtS7IbPrx5505Uw3X3V8tauviePfrc=";
+  };
```

Comment:
Have you tried using the GH repo instead of Pypi ?

Reply:
I have not, is there a recommendation to go with github instead of pypi?

Reply:
I actually have to go with github to get tests. Tests are stripped off from the dist package that is made available on pypi.